bundles/model/org.eclipse.smarthome.model.rule/src/org/eclipse/smarthome/model/rule/Rules.xtext

```diff
@@ -72,6 +74,13 @@ SystemOnShutdownTrigger:
 	'System' 'shuts down'
 ;
 
+ThingStateUpdateEventTrigger:
+    'Thing' thing=STRING 'received update' (state=ValidState)?
```

Comment:
The update is not a "ValidState", but ThingStatus (or even rather a ThingStatusInfo, check out the classes - it would need to be decided what syntax to use here in the DSL for it).